File: Laborator/Lab 4/JavaClientServer/Networking/src/main/java/utils/Parser.java
package utils;

import java.util.LinkedList;
import java.util.List;

public class Parser {
    public static String ToString(int idEv,int ageMin,int ageMax)
    {
        return idEv+" "+ageMin+" "+ageMax;
    }
    public static List<Integer> ToIntList(String str)
    {
        String[] l=str.split(" ");
        List<Integer> li=new LinkedList<>();
        li.add(Integer.parseInt(l[0]));
        li.add(Integer.parseInt(l[1]));
        li.add(Integer.parseInt(l[2]));
        return li;
    }
}

File: Laborator/Lab 4/C# Client-server/Networking/Utils/Parser.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Networking.Utils
{
    public class Parser
    {
        public static string ToString(int idEv, int ageMin, int ageMax)
        {
            return idEv + " " + ageMin + " " + ageMax;
        }

        public static IList<int> ToList(string str)
        {
            string[] fields = str.Split(' ');
            IList<int> result = new List<int>();

            result.Add(Int32.Parse(fields[0]));
            result.Add(Int32.Parse(fields[1]));
            result.Add(Int32.Parse(fields[2]));

            return result;
        }
    }
}
